app/views/records/_record.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class='record d-flex flex-center' >
+  <p>
+    <strong><%= record.name %></strong>
+  </p>
+
+  <p>
+  <strong><span><%= record.amount %><span> $</strong>
+  </p>
+</div>
```

Comment:
- [ ]  Here we need to see when the transaction happened. Please mention the date when it is created.

### Your Design

![Screenshot from 2023-02-25 21-07-38](https://user-images.githubusercontent.com/26767383/221366338-ba1a43fe-4ca0-403b-8aee-ca1155c992a2.png)


### Given design

![Screenshot from 2023-02-25 04-42-14](https://user-images.githubusercontent.com/26767383/221318942-7184a138-6a9c-4085-9410-56d70dbb766e.png)

Reply:
@Deepakdanger, This was a great observation, I had not realized it and you made me see a few things I had forgoten, I know understand why it is important for a developer to review my code